debian/changelog

```diff
@@ -1,3 +1,16 @@
+cloud-init (24.1.3-0ubuntu3.2) noble; urgency=medium
```

Comment:
Good versioning FWIW as it's the third published upload of back-ported content to noble